src/content/language/extension-types.md

```diff
@@ -446,8 +446,9 @@ It's important to be aware of this quality when using extension types.
 Always keep in mind that an extension type exists and matters at compile time,
 but gets erased _during_ compilation.
 
-An expression with the extension type `E` and the representation type `R`
-as its static type will be an object with type `R` at run time.
+For example, consider an expression `e` whose static type is the
+extension type `E`, and the representation type of `E` is `R`.
+Then, the run-time type of the value of `e` is a subtype of `R`.
```

Comment:
@eernstg I think you said it would be "OK" to say `e`'s type is R, so could we maybe combine the two points just to be extra explicit?:
```suggestion
Then, the run-time type of the value of `e` is `R` or a subtype of `R`.
```

Reply:
It's always tricky to try to predict the interpretation that "most readers" will use for any given wording. For example, "the object `o` has type `R`" could mean "the run-time type of `o` is `R`" or it could mean "the run-time type of `o` is a subtype of `R`", and the latter _always_ means "the run-time type of `o` is `R` or a subtype of `R`".

So should we go over all of site-www and change "is a subtype of `T`" to "is `T` or a subtype of `T`", or should we try to say, once and for all, that the subtype relation is reflexive and "a subtype of `T`" _always_ includes `T` itself? (The specification documents, and perhaps some other documents, use 'a proper subtype of `T`' to denote that rare exception where `T` is not included.)

I think that's a sitewide policy matter, so I'll just rely on you knowing much more about which approach is preferred globally.

Reply:
Aha, I did not realize that subtypes are a reflexive relation and was just speaking colloquially (like, for the sake of overt clarity). If that's the definition, then I think it's safe to use it correctly! I'll scrap the suggestion